examples/kotlin/hello_world/MainActivity.kt

```diff
@@ -36,7 +37,9 @@ class MainActivity : Activity() {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_main)
 
-    streamClient = AndroidStreamClientBuilder(application).build()
+    streamClient = AndroidStreamClientBuilder(application)
+      .addFilter({ DemoFilter() })
```

Comment:
```suggestion
      .addFilter { DemoFilter() }
```

You don't have to add the parens for this case